### ISSUE TITLE
Object Detection API: Batch Normalization not applied on BoxPredictor (SSD)

### ISSUE THREAD

**Issue by wmouchere** [tensorflow/models#2237]
It seems batch normalization and custom activation function are not supported for the classification and regression layers for the SSD architecture. I found this line : https://github.com/tensorflow/models/blob/b14765c3a8b119ab8dae3d706d623397f99d1011/object_detection/core/box_predictor.py#L515 and successfully added batch norm by replacing it with `with slim.arg_scope(self._conv_hyperparams), slim.arg_scope([slim.conv2d], activation_fn=None):`.
Not sure if this is intended but the MobileNet Pets configuration has batch normalisation in the box predictor config, which is misleading !
(See  https://github.com/tensorflow/models/blob/b14765c3a8b119ab8dae3d706d623397f99d1011/object_detection/samples/configs/ssd_mobilenet_v1_pets.config#L72)

**Comment by skye** [comment]
@jch1 @tombstone @derekjchow @jesu9 @dreamdragon

**Comment by shamanez** [comment]
Box Predictor is predicting class scores and location coordinates not in a fully connected manner . If there are 4 classes including background  , we will have 4+4(coordinates) = 8 filters looking at one position in the feature map.  So basically there are no any kind of non linear units. (RELU_6 = None). 

Convolution hyper parameters for the boxpredictor are mostly used in when we need to add additional layers before the prediction .

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
